aspnetcore/blazor/fundamentals/additional-scenarios.md

```diff
@@ -116,24 +116,105 @@ Rendering server components from a static HTML page isn't supported.
 
 *This section applies to Blazor Server.*
 
-Sometimes, you need to configure the SignalR client used by Blazor Server apps. For example, you might want to configure logging on the SignalR client to diagnose a connection issue.
+Configure the SignalR client used by Blazor Server apps in the `Pages/_Host.cshtml` file. Place a script that calls `Blazor.start` after the `_framework/blazor.server.js` script and inside the `</body>` tag.
 
-To configure the SignalR client in the `Pages/_Host.cshtml` file:
+### Logging
+
+To configure SignalR client logging:
 
 * Add an `autostart="false"` attribute to the `<script>` tag for the `blazor.server.js` script.
-* Call `Blazor.start` and pass in a configuration object that specifies the SignalR builder.
-
-```html
-<script src="_framework/blazor.server.js" autostart="false"></script>
-<script>
-  Blazor.start({
-    configureSignalR: function (builder) {
-      builder.configureLogging("information"); // LogLevel.Information
-    }
-  });
-</script>
+* Pass in a configuration object (`configureSignalR`) that calls `configureLogging` with the log level on the client builder.
+
+```cshtml
+    ...
+
+    <script src="_framework/blazor.server.js" autostart="false"></script>
+    <script>
+      Blazor.start({
+        configureSignalR: function (builder) {
+          builder.configureLogging("information");
+        }
+      });
+    </script>
+</body>
 ```
 
+In the preceding example, `information` is equivalent to a log level of <xref:Microsoft.Extensions.Logging.LogLevel.Information?displayProperty=nameWithType>.
+
+### Modify the reconnection handler
+
+To modify the reconnection handler's circuit connection events:
```

Comment:
Let's add some more content here to showcase why the user might need to do this. Specifically, if they want to notify the user in some custom way if the connection is dropped, if they want to do some logging (from the client) when a circuit is connected, etc.